src/index.html

```diff
@@ -9,7 +9,103 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
+  <form method="post" action="https://mate.academy/ru">
+    <fieldset>
+      <legend>Personal information</legend>
+      <div class="surname">
+        <label>Surname:
+          <input id="surname" type="text" name="surname" value="" required>
```

Comment:
```suggestion
          <input id="surname" type="text" name="surname" required>
```
You do not need to specify a default value for inputs. There is nothing wrong with that, but it is not necessary to write.

Reply:
done